app/serializers/serialize_track.rb

```diff
@@ -18,10 +18,7 @@ def call
       icon_url: track.icon_url,
       tags: map_tags(track.tags),
       last_touched_at: user_track.last_touched_at&.iso8601,
-
-      # TODO: Set these
-      is_new: true,
-
+      is_new: track.created_at > Time.zone.now - 6.months,
```

Comment:
I normally use `Time.current` here instead of `Time.zone.now` - probably worth changing this to be consistent.